src/sql-parser/tests/testdata/create

```diff
@@ -215,3 +215,10 @@ CREATE TABLE "".table_name (col_name int)
 error: zero-length delimited identifier
 CREATE TABLE "".table_name (col_name int)
              ^
+
+parse-statement
+CREATE TABLE " " (x int)
```

Comment:
There is a null byte between the quotes. GitHub's diff UI just refuses to show it.

Reply:
What it looks like in my editor:
![image](https://user-images.githubusercontent.com/1893159/222528497-d7c691d7-4a58-497c-8267-73b61dd94fcd.png)